example/broadcast_server.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+import asyncio
+import time
+import websockets
+
+async def broadcast():
+    while True:
+        message = str(time.time())
+        await asyncio.gather(
+            *[ws.send(message) for ws in CLIENTS],
+            return_exceptions=False,
```

Comment:
Haven't tested this, but IMHO there should be `return_exceptions=True`, as from the docs: "exceptions are ... aggregated in the result list".  (We probably do not want to see `ConnectionClosed` exceptions propagated into `broadcast()`.)

Reply:
Not sure, I think it is this way so that exceptions immediately propagate into `handler` which removes the corresponding websocket from the `CLIENTS` set. However it has been a while and my understanding of `asnycio` is poor. My testing (as I recall) consisted of running several clients and verifying that if I killed one with CTRL-C everything else kept running as expected.